src/ocm/components/clusterWizard/ClusterDetailsForm.tsx

```diff
@@ -148,7 +148,7 @@ const ClusterDetailsForm = (props: ClusterDetailsFormProps) => {
                 (dirty || (cluster && canNextClusterDetails({ cluster })))
               )
             }
-            onNext={handleOnNext(dirty, submitForm, cluster)}
+            onNext={void handleOnNext(dirty, submitForm, cluster)}
```

Comment:
It may not be due to this change, but in OCM the  "Next" button is missing.
I wanted to check if it worked fine because I think in this case the return value is needed.
![next-button](https://user-images.githubusercontent.com/829045/211283327-bd358f44-134d-47aa-b41d-fabdf645e3b1.png)

Reply:
I don't think the return value is used. Plus it is triggered synchronously without waiting on result.

TypeScript:
```export type WizardFooterGenericProps = {
  onNext?: () => void;
  ```
  
  Used as-it-is for Button's `onClick`

Reply:
Looks right, I didn't check but there is definitely a problem with the Next button. Could you see what triggered it?